src/SelfGravity/Poi_UserWorkBeforePoisson_GREP.cpp

```diff
@@ -25,7 +25,6 @@ Profile_t *GREP_EffPot  [NLEVEL  ][2];
 int    GREP_LvUpdate;
 int    GREP_Sg     [NLEVEL];
 double GREP_SgTime [NLEVEL][2];
-double GREP_Center [3];
```

Comment:
Remove the declaration of `GREP_Center` in other places since it is now a global variable.

Reply:
I removed the declaration of `GREP_Center` in other places.